employee-rostering-backend/src/main/java/org/optaweb/employeerostering/service/roster/RosterGenerator.java

```diff
@@ -418,13 +418,6 @@ public void checkForExistingData() {
 
     @Transactional
     public void setUpGeneratedData() {
-        // Check if Tenant entities already exist before generating data
```

Comment:
Why did you delete this? This ensures that the data is not regenerated if you reboot the application with a persistent database.

Reply:
This check was already done in the `checkForExistingData()` method above, which is called when the application starts.